plugins/apps/threatbus_cif3/message_mapping.py

```diff
@@ -0,0 +1,67 @@
+from threatbus.data import Intel, IntelType, Operation
+from csirtg_indicator import Indicator
+from csirtg_indicator.exceptions import InvalidIndicator
+
+cif_supported_types = [
+    IntelType.IPSRC,
+    IntelType.IPDST,
+    IntelType.EMAILSRC,
+    IntelType.HOSTNAME,
+    IntelType.DOMAIN,
+    IntelType.URL,
+    IntelType.MD5,
+    IntelType.SHA1,
+    IntelType.SHA256,
+    IntelType.AUTHENTIHASH,
+    IntelType.SSDEEP,
+    IntelType.IMPHASH,
+    IntelType.PEHASH,
+]
+
+
+def map_to_cif(intel: Intel, logger=None, config=None):
```

Comment:
```suggestion
def map_to_cif(intel: Intel, logger, confidence, tags, tlp, group):
```

This function should not receive the config as arguemtn, but rather the required variables (`confidence`, `tags`, `tlp`, and `group`). Also, `logger` should not be initialized with `None` since it is used below in exception handling.